.golangci.yml

```diff
@@ -21,7 +24,6 @@ linters:
     - godot
     - gofmt
     - goimports
+    - gosimple
```

Comment:
Thank you for doing this!
Can you help me understand the below?
Are most of the changes in this PR from an automatic cleanup (gosimple) ?
Do the .golangci.yml file changes specify that a gosimple cleanup will automatically be run on all code when a PR is  generated?